components/brave_vpn/common/buildflags/BUILD.gn

```diff
@@ -3,10 +3,50 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
+import("//brave/build/config.gni")
 import("//build/buildflag_header.gni")
 import("buildflags.gni")
 
 buildflag_header("buildflags") {
   header = "buildflags.h"
   flags = [ "ENABLE_BRAVE_VPN=$enable_brave_vpn" ]
+
+  if (is_win) {
```

Comment:
there's really no reason for the vpn helper to be in components in the first place since it's win specific so it can just use `install_static::GetChromeChannel`
plus there are already layering violations in here with `install_static`

Reply:
Filed https://github.com/brave/brave-browser/issues/35260 and working on it